### DISCUSSION TITLE
Multi-Language UI support

### DISCUSSION THREAD

**Discussion by adamsmodi** [TabularEditor/TabularEditor3#1099] [Ideas]
Hello Tabular Editor 3 team,

My name is Mo Di and have been using Tabular Editor since years, big fan, thanks a lot. I come from China and now work in Germany. We are having a bigger and bigger user group (>1000) learning together the efficient functionality in Tabular Editor.. 
Meanwhile, we are also supporting Power BI project implementation for a lot of companies in China. There is also Microsoft MVP
The problem for us is that the language settings in Tabular Editor is only English and for big models and Chinese local company, it would be difficult to launch.

I have considered this function since long time. Recently, I tried to support the UI translation for the newly launched Bravo.bi tool from SQLBI team and it worked fine. That encourage me a little bit to check with Tabular Editor team for the potential as well.
My native language is Chinese and I am fluent in English and German. In case that you will consider this funtion in the future, I want to volunteer myself for supporting Chinese and German translation..

thanks a lot for your consideration in advance.

**Comment by otykier** [comment]
Same answer as [here](https://github.com/TabularEditor/TabularEditor/issues/988). Thanks again for volunteering - we will reach out once UI localization is possible.

**Comment by mlonsk** [comment]
Since this spring Tabular Editor 3 is avaliable in multiple languages with Chinese being one of them. Thank you for your patience while we to this work done.
